clash-prelude/src/Clash/Explicit/Signal.hs

```diff
@@ -46,8 +46,8 @@ made. Clash provides a standard implementation, called 'System', that is
 configured as follows:
 
 @
-instance KnownDomain "System" where
-  type KnownConf "System" = 'DomainConfiguration "System" 10000 'Rising 'Asynchronous 'Defined 'ActiveHigh
+instance KnownDomain 'System' where
+  type KnownConf 'System' = 'DomainConfiguration 'System' 10000 'Rising 'Asynchronous 'Defined 'ActiveHigh
```

Comment:
Is this what we want? It could either refer to the type synonym `System` or to the type-level string `"System"`, I believe.  They are the same thing. So it is either this or
```
instance KnownDomain \"System\" where
  type KnownConf \"System\" = 'DomainConfiguration \"System\" 10000 'Rising 'Asynchronous 'Defined 'ActiveHigh
```